samples/meetings-attendance-report/csharp/README.md

```diff
@@ -0,0 +1,157 @@
+---
+page_type: sample
+description: This is a sample application which demonstrates how to get meeting attendance report using Graph API and send it in meeting chat.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+contentType: samples
+createdDate: "20-08-2022 00:01:15"
+---
+
+# Bot to show the transcript of the meeting using Microsoft Graph API.
+
+This is a sample application which demonstrates how to get meeting attendance report using Graph API and send it in meeting chat.
+
+## Key features
+
+When meeting end attendance report card is sent by the bot
```

Comment:
When meeting ends, attendance report card is sent by the bot.

Reply:
Done